header-footer-grid/Core/Builder/Footer.php

```diff
@@ -75,6 +75,26 @@ public function init() {
 		$this->devices = [
 			'desktop' => __( 'Footer', 'neve' ),
 		];
+
+		/**
+		 * Fix legacy search widget display in footer
+		 * Only applies for versions < 5.8
+		 */
+		add_filter(
+			'dynamic_sidebar_params',
+			function ( $params ) {
+				$processed_params        = [];
+				$has_legacy_search_style = apply_filters( 'neve_has_legacy_search_style_filter', false );
+				foreach ( $params as $param ) {
+					if ( isset( $param['widget_name'] ) && $param['widget_name'] === 'Search' && $has_legacy_search_style === false ) {
```

Comment:
@cristian-ungureanu I've changed the check so that is language independent.
@irinelenache can you also check that only one instance of `.widget_search .search-form .search-submit, .widget_search .search-form .search-field { height: auto; }` is present on the page if multiple legacy search widgets are present?

Thank you.